app/views/proposals/_proposal.html.erb

```diff
@@ -86,6 +86,27 @@
                             <% end %>
                           </div>
                         </div>
+                        </div>
+                        <div>
+                          <%= label_tag :assigned_location_id %>
+                        </div>
+                        <div class="mb-3">
+                          <%= f.text_field :assigned_location_id, class: 'form-control w-25', disabled: true, value:@proposal.assigned_location&.code %>
+                        </div>
+                         <div>
+                          <%= label_tag :assigned_date %>
+                        </div>
+                        <div class="mb-3">
+                          <%= f.text_field :assigned_date, class: 'form-control w-25', disabled: true, value:@proposal.assigned_date %>
+                        </div>
```

Comment:
I'm not sure a text field will work for a `Date` object. Perhaps a select menu showing available dates between `Location.start_date` and `Location.end_date`, displayed as `start_date.strftime("%Y-%m-%d")`.